<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCapture</class>
 <widget class="QDialog" name="VideoCapture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>646</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>201</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="deviceCB"/>
    </item>
    <item>
     <widget class="QComboBox" name="configCB"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QFormLayout" name="currentConfig_FLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="form_brightnessLabel">
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="form_brightnessSlider">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="form_contrastLabel">
        <property name="text">
         <string>Contrast</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="form_contrastSlider">
        <property name="minimum">
         <number>-8</number>
        </property>
        <property name="maximum">
         <number>64</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="form_zoomLabel">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="form_zoomSlider">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="form_framerateLabel">
        <property name="text">
         <string>Framerate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="form_framerateSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="pagination">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>401</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_video">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QLabel" name="video_displayLabel">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>12</y>
       <width>361</width>
       <height>231</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NO DEVICE SELECTED</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_config">
    <attribute name="title">
     <string>Config</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="GLayout_2_1">
      <item row="1" column="1">
       <widget class="QComboBox" name="config_pickCB"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="config_nameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="config_setBtn">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="config_addBtn">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="config_defaultDeviceBtn">
        <property name="text">
         <string>Make</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="config_defaultConfigBtn">
        <property name="text">
         <string>Make</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="config_infoLabel_1">
        <property name="text">
         <string>current Device as Default</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="config_infoLabel_2">
        <property name="text">
         <string>current Config as Default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoCapture</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
